integration-tests/jail/disk/config0/config.toml

```diff
@@ -0,0 +1,319 @@
+# This is a TOML config file.
```

Comment:
could this setup be generated instead of hardcoded

Reply:
may be useful to incorporate the chain bot @yihuang wrote

Reply:
it's possible, but scripts will be complex.
i'm thinking to make genesis generation directly by dev-conf.json directly.
i'll find a simpler way

Reply:
i'll find a way to simplify this.

Reply:
i'll check yiyuang's script